universe/marathon.json.mustache

```diff
@@ -41,38 +41,25 @@
       {"port": {{networking.agent-port}}, "protocol": "tcp", "name": "agent"}
   ],
   "container": {
-        "type": "{{advanced.containerizer}}",
```

Comment:
Why are changing this to use only MESOS container type?

Reply:
Reverted this changes. as default containerizer was set to Docker